package.json

```diff
@@ -224,7 +224,9 @@
     "@elastic/eslint-import-resolver-kibana": "1.0.0",
     "@elastic/eslint-plugin-kibana-custom": "link:packages/eslint-plugin-kibana-custom",
     "@kbn/plugin-generator": "link:packages/kbn-plugin-generator",
+    "@kbn/plugin-helpers": "link:packages/kbn-plugin-helpers",
     "angular-mocks": "1.4.7",
+    "ansicolors": "0.3.2",
```

Comment:
Move to chalk at the same time as we're moving this into Kibana? (as we've already moved away from `ansicolors` in Kibana: https://github.com/elastic/kibana/issues/11708)